niri-config/src/lib.rs

```diff
@@ -881,13 +889,16 @@ pub enum Action {
     FocusWorkspaceDown,
     FocusWorkspaceUp,
     FocusWorkspace(#[knuffel(argument)] u8),
+    FocusNamedWorkspace(#[knuffel(argument)] String),
```

Comment:
Unfortunate that these have to be separate, but this is more of a knuffel limitation... And having to write custom Decode impl for this would be pretty annoying. Although this isn't the first time knuffel prevented me from doing something here, so maybe it'll come down to this at some point.

Reply:
I ended up writing a custom `DecodeScalar` impl for a `pub enum WorkspaceReference` =)